deploy/examples/keycloak_min.json

```diff
@@ -8,6 +8,7 @@
     ]
   },
   "spec": {
-    "adminCredentials": ""
+    "adminCredentials": "",
+    "plugins": "keycloak-metrics-spi"
```

Comment:
Should we keep the minimum example more bare bones, and not have any plugins listed?